### ISSUE TITLE
Package Request: python27-libxml

### ISSUE THREAD

**Issue by mike-pt** [Entware-ng/Entware-ng#201]
Its not included in the python package and unfortunately pip failed to build it, would be awesome to have it

**Comment by zyxmon** [comment]
```
opkg list | grep xml
....
python-xml - 2.7.11-3 - Python 2.7 xml libs
....
```

**Comment by mike-pt** [comment]
I made a mistake while typing this yesterday, I edited the title after but it seems something failed.

I meant to request lxml which is a different thing :)

**Comment by mike-pt** [comment]
could this be considered for re-opening? as stated I'm made a typo on the initial submission its "lxml" not libxml

**Comment by zyxmon** [comment]
Since many perl/python modules can be built by users it a is low priority issue.

**Comment by jebeaudet** [comment]
Just so others might find a solution, I've managed to build lxml on armv7. After installing gcc and the headers as described here https://github.com/Entware-ng/Entware-ng/wiki/Using-gcc-%28native-compilation%29, I just had to make a symbolic link for the libxml headers as they were not in the expected place : 
```
ln -s /opt/include/libxml2/libxml/ /opt/include/libxml
```
This should do it. Hopefully I'm not breaking any rules commenting on a closed thread, I just figured others might gain from my experience. Thanks for your work!

https://github.com/Entware-ng/Entware-ng/issues/218 is a duplicate.
